php/libraries/NDB_Config.class.inc

```diff
@@ -414,6 +414,22 @@ class NDB_Config
         return $this->getSettingFromDB($name);
     }
 
+    /**
+     * Helper function to determine whether a setting with a boolean value is
+     * enabled, i.e. equal to "true".
+     * Since settings are returned as strings, this must be done as a string
+     * comparison.
+     *
+     * @param string $name The name of the setting to retrieve. This setting
+     *                      should have a boolean value in the config file or DB.
+     *
+     * @return bool True if the setting is equal to "true".
+     */
+    function settingEnabled(string $name): bool
+    {
+        return $this->getSetting($name) === 'true';
```

Comment:
```suggestion
        $val = $this->getSetting($name);
        return $val === 'true' || $val === '1';
```

If adding a helper, might as well make it robust. This should also be used somewhere and not just added for the sake of being added.